feat(llc): added mirror parameter to CameraConstraints (#641)
GetStream/stream-video-flutter: packages/stream_video/CHANGELOG.md

## Unreleased
✅ Added

> * `mirror` parameter to `CameraConstraints`, that is used to determine if the camera for the given track should be mirrored or not. When set as null the default is true when f`acingMode` is set to `user` and false when facingMode is set to `environment`.

Reviewer: ```suggestion
* Added `mirror` parameter to `CameraConstraints`, which determines whether the camera for the given track should be mirrored or not. When set as null the default is true when `facingMode` is set to `user` and false when facingMode is set to `environment`.
```
